app/src/pages/trace/SpanAside.tsx

```diff
@@ -1,13 +1,58 @@
-import React, { PropsWithChildren } from "react";
+import React, { createContext, useCallback, useContext, useState } from "react";
 import { graphql, useRefetchableFragment } from "react-relay";
 import { css } from "@emotion/react";
 
-import { Flex, Text, View } from "@phoenix/components";
+import {
+  Flex,
+  Tab,
+  TabList,
+  TabPanel,
+  Tabs,
+  Text,
+  View,
+} from "@phoenix/components";
 import { AnnotationLabel } from "@phoenix/components/annotation";
+import { SpanAnnotationsEditor } from "@phoenix/components/trace/SpanAnnotationsEditor";
 
 import { SpanAside_span$key } from "./__generated__/SpanAside_span.graphql";
 import { SpanAsideSpanQuery } from "./__generated__/SpanAsideSpanQuery.graphql";
 
+export const SpanAsideContext = createContext<{
+  tab: "feedback" | "annotate";
+  setTab: (tab: string) => void;
+}>({
+  tab: "feedback",
+  setTab: () => {},
+});
+
+export const useSpanAsideState = (
+  defaultTab: "feedback" | "annotate" = "feedback"
+) => {
+  const [tab, _setTab] = useState<"feedback" | "annotate">(defaultTab);
```

Comment:
I was thinking rather than tabs, we would just replace part of the content. Tabs obscure content so wanted to avoid hiding stuff. I was thinking Annotate button would then be a toggle button.